index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NoteBook</title>
+    <style>
+        *{
+            margin: 0;
+            padding:0;
+            box-sizing: border-box;
+        }
+
+        body{
+            min-height: 100vh;
+            background:#d3d3d3;
+            font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+            display:flex;
+            justify-content:center;
+            align-items:center;
+        }
+        ::-webkit-scrollbar{
+            display: none;
+        }
+        .paper{
+            position:relative;
+            width: 90%;
+            max-width:800px;
+            min-width:400px;
```

Comment:
Are you using Prettier VsCode Extension?, It will automatically format your code, As there is always a space after colan ":" eg: max-width: 800px;

Reply:
Yes you should obviously use. It will save your time in formatting your code. What I was saying is that check in your code, there is space after colan, and at some places, there isn't.  Prettier automatically gives a space after the colan. Try reinstalling your extension or check in it's setting. Code should look clean and uniform. Hope you understood my concern.

Reply:
yes.